modules/rosa-create-objects.adoc

```diff
@@ -768,6 +771,13 @@ Add a machine pool with labels to a cluster.
 $ rosa create machinepool --cluster=mycluster --replicas=2 --instance-type=r5.2xlarge --labels=foo=bar,bar=baz --name=mp-1
 ----
 
+Add a machine pool with tags to a cluster.
+
+[source,terminal]
+----
+$ rosa create machinepool --cluster=mycluster --replicas=2 --instance-type=r5.2xlarge --tags=foo:bar,bar:baz --name=mp-1
```

Comment:
Let's show example follow help message hint.

Reply:
Good catch - I changed it to match like so:
![image](https://github.com/openshift/openshift-docs/assets/16167833/671a2582-0eba-4ae1-b1e4-6255fd0bd0a7)